disttask/framework/scheduler/manager.go

```diff
@@ -218,6 +235,22 @@ func (m *Manager) onCanceledTasks(_ context.Context, tasks []*proto.Task) {
 	}
 }
 
+// onPausingTasks pauses/cancels the pending/running tasks.
+func (m *Manager) onPausingTasks(tasks []*proto.Task) error {
+	m.mu.RLock()
```

Comment:
Why here add lock? will there concurrent access manager? I thought manager only one for one TiDB.

Reply:
fetchAndHandleRunnableTasks and fetchAndFastCancelTasks use 2 goroutines  to access handlingTasks.